modules/samtools/merge/main.nf

```diff
@@ -8,8 +8,9 @@ process SAMTOOLS_MERGE {
         'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
 
     input:
-    tuple val(meta), path(input_files)
+    tuple val(meta), path(input_files, stageAs: "?/*")
```

Comment:
why is the stageAs needed here?

Reply:
I have a case where my input cram files possibly can have the same file name. This fixes that issue without influencing the way the module works for other people